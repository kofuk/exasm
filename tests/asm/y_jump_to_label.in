j @L1
nop
@L1 nop
