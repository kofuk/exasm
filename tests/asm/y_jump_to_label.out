@00 11000000 00000010 // j 0x02
@02 00000000 00000000 // nop
@04 00000000 00000000 // nop
